libs/ui-watt/src/lib/foundations/_variables.scss

```diff
@@ -0,0 +1,60 @@
+@use './color/colors.import' as colors;
```

Comment:
Suggestion: Remove this import since it's no longer necessary.

Suggestion: Move this file under `color` directory and rename it to `index.scss` (like in Typography). So the import in `watt.scss` will become:

```
@use './../../foundations/color';
```

Question: Do you think it's better if we rename the "color" folder to "colors" (plural). It sounds more correct when looking from outside since there are multiple colors defined in it.